rust/core-lib/src/event_context.rs

```diff
@@ -160,6 +160,12 @@ impl<'a> EventContext<'a> {
             Edit { edit } => self.with_editor(
                 |ed, _, _| ed.apply_plugin_edit(edit)),
             Alert { msg } => self.client.alert(&msg),
+            AddStatusItem { key, value, alignment }  =>
+            self.client.add_status_item(viewID, &key, &value, &alignment),
+            UpdateStatusItem { key, value }
+            => self.client.update_status_item(viewID, &key, &value),
```

Comment:
rust style nit: the `=>` should be on the same line as the match case; and if the statement for that case is on a newline it should be indented.